### ISSUE TITLE
Update Dart SDK constraint in pubspec.yaml

### ISSUE THREAD

**Issue by kherel** [yakir8/kosher_dart#46]
The package currently restricts the Dart SDK version to ">=3.0.0 <3.5.0". With the Dart SDK now beyond version 3.5.0, this constraint prevents the package from being installed with the latest SDK. Since the code doesn't seem to require this restriction, please update the constraint to support newer SDK versions, such as ">=3.0.0 <4.0.0".

**Comment by mendelg** [comment]
As of now, to solve the issue, you can override the dependency in your `pubspec.yaml`:




```
dependency_overrides:
  kosher_dart:
    git:
      url: https://github.com/yakir8/kosher_dart.git
      ref: master
```

**Comment by yakir8** [comment]
close

fixed on kosher_dart 2.0.17
